hoot-core-test/src/test/cpp/hoot/core/ops/CalculateStatsOpTest.cpp

```diff
@@ -80,7 +80,7 @@ class CalculateStatsOpTest : public HootTestFixture
   void runStatsNumTest()
   {
     std::shared_ptr<CalculateStatsOp> calcStatsOp = _calcStats(_inputPath + "all-data-types.osm");
-    CPPUNIT_ASSERT_EQUAL(172, calcStatsOp->getStats().size());
+    CPPUNIT_ASSERT_EQUAL(193, calcStatsOp->getStats().size());
```

Comment:
All the stat count changes in this test are due to the new matchers being added.